### DISCUSSION TITLE
oo/package

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#279] [Announcements]
# oo/package

Java中的包是一种用于组织和管理代码的机制，可以帮助开发者更有效地处理复杂的项目。本文将详细讲解Java包的概念、创建、导入以及使用方法，以及如何通过包提高代码可读性和模块化。掌握Java包的使用技巧，为更高级的Java编程打下坚实基础。

https://javabetter.cn/oo/package.html

**Comment by BOUNDLESS-558** [comment]
稍补充一点，除了 "private" 、 "public" 、 "protected" 和 "默认权限修饰符（什么都不写）"外，自 JDK 8 开始还新增了一个 "default" 关键字可以用来在接口中声明默认方法，这是比较容易引起混淆的地方。例如：

```java 
 interface Game {
     // default 不是默认权限修饰符，什么都不写才是。
     default void startGame() {
         System.out.println("游戏启动");
     }
 }
```
